app/src/main/java/com/kickstarter/viewmodels/ProjectSearchResultHolderViewModel.java

```diff
@@ -0,0 +1,122 @@
+package com.kickstarter.viewmodels;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.util.Pair;
+
+import com.kickstarter.libs.ActivityViewModel;
+import com.kickstarter.libs.Environment;
+import com.kickstarter.libs.utils.ProjectUtils;
+import com.kickstarter.models.Photo;
+import com.kickstarter.models.Project;
+import com.kickstarter.ui.viewholders.ProjectSearchResultViewHolder;
+
+import rx.Observable;
+import rx.subjects.BehaviorSubject;
+import rx.subjects.PublishSubject;
+
+import static com.kickstarter.libs.rx.transformers.Transformers.takeWhen;
+
+public interface ProjectSearchResultHolderViewModel {
+
+  final class Data {
+    final Project project;
+    final boolean isFeatured;
```

Comment:
feel free to leave as is then! 

FWIW we do use Pairs quite a bit in our view models, and while it's not always clear what .first and .second always are, we try to use descriptive variable names (e.g. `projectAndUser`) so that at the least `projectAndUser.first` can tell us that that value is a Project